wear/src/main/java/io/homeassistant/companion/android/home/views/SettingsView.kt

```diff
@@ -105,6 +107,29 @@ fun SettingsView(
                         onClick = onClearFavorites
                     )
                 }
+                item {
+                    ToggleChip(
+                        modifier = Modifier.fillMaxWidth(),
+                        checked = isFavoritesOnly,
+                        onCheckedChange = { setFavoritesOnly(it) },
+                        label = { Text(stringResource(commonR.string.only_favorites)) },
+                        toggleControl = {
+                            Icon(
+                                imageVector = ToggleChipDefaults.checkboxIcon(isFavoritesOnly),
```

Comment:
I'm not sure a checkbox is the appropriate control here, as those are generally used for [one or more items from a set](https://m3.material.io/components/checkbox/overview). For feedback options it's a set but here it's [on or off, standalone](https://m3.material.io/components/switch/overview).